tests/FunctionalTest.php

```diff
@@ -191,8 +193,9 @@ private function createObjectManager(): ObjectManager
 
         $objectPersisterFactory = new ObjectPersisterFactory();
 
+        /** @var \Doctrine\Persistence\Mapping\ClassMetadataFactory<ClassMetadata<object>> $classMetadataFactory */
```

Comment:
@greg0ire The excluded errors I mentioned in Skeleton-Mapper https://github.com/doctrine/skeleton-mapper/pull/39#issuecomment-1372879914 are affecting this line in this test. Any idea now how it can be fixed?

Reply:
I thought I reproduced the bug: https://phpstan.org/r/b486420d-aadb-4799-b283-271fe1b21efe

Yet I can't seem to be able to apply this fix: https://phpstan.org/r/f8da4243-2585-4ba0-9a03-203c8a90fd76

Reply:
One is the ClassMetadataFactory and ClassMetadata of Persistence and the others of SkeletonMapper they're not interchangeable, but SkeletonMapper extends the ones of Persistence and so the differences of types are causing the phpstan issues. It's something that needs to be fixed in SkeletonMapper first.